packages/vats/src/types.d.ts

```diff
@@ -89,13 +89,13 @@ export type NamesByAddressAdmin = NameAdmin & {
 /** An object that can receive messages from the bridge device */
 export type BridgeHandler = {
   /** Handle an inbound message */
-  fromBridge: (obj: any) => Promise<void>;
+  fromBridge: (obj: any) => PromiseVow<void>;
```

Comment:
Is this a breaking change to the bridge vat interface? Are we upgrading the bridge vat and all its clients?

Reply:
Or maybe the interface can sort of vary by scoped bridge handler?
I expect expressing that as a static type could be hairy, but as an upgrade plan, maybe it works?

Reply:
As an alternative to Option 2, we could do something like this: https://github.com/Agoric/agoric-sdk/pull/8964/commits/b80ea7c3ceb9883144b7082745019c8bb865eee6, where we keep the `BridgeHandler` interface the same, but take advantage of `V`

Reply:
`PromiseVat<void>` is just `Promise<void | Vow<void>>`, which reduces to `Promise<void>`.  So you could have option 0:
0. Don't change anything in this PR, and just make a note that "if a given `fromBridge` returns an actual `Vow`, we will need to accommodate that in the `bridge` vat someday."